README.md

```diff
@@ -11,27 +11,69 @@ Reviewcheck is in active development.
 
 ## Installation
 
-While the project is still in development, the best way to install it is by
-cloning the repository and running `poetry run reviewcheck` from within it. You
-will need to have poetry installed. The process looks as follows:
+Reviewcheck can be installed with the following command:
 
+```Shell
+pip install reviewcheck
 ```
+
+If you are a developer, the best way to test changes you make without having to
+create a Python package and install it is to clone the repository and run
+`poetry run reviewcheck` from within it. You will need to have poetry
+installed. The process looks as follows:
+
+```Shell
 pip install poetry
 git clone https://github.com/volvo-cars/Reviewcheck
 cd reviewcheck
 poetry run reviewcheck
 ```
 
-## Documentation
-
-For now, the only documentation is this README and the `--help` flag of the
-program itself. A proper
-[Wiki](https://github.com/volvo-cars/Reviewcheck/wiki) is
-under construction.
-
-## FAQ [NYI]
-
-See [FAQ (NYI)](docs/faq/faq.md) for more questions.
+## Getting Started
+
+Reviewcheck is a TUI app. You run it on the command line with `reviewcheck`.
+The first time you run it, you will be able to interactively configure it to
+find your reviews.
+
+After that, any time you run `reviewcheck`, you will be presented with a list
+of threads from any MR where your response is needed. You can configure how
+verbose the output is with options. See the `--help` option for more information
+about that.
+
+## FAQ
+
+<dl>
```

Comment:
Could this HTML be replaced with markdown? E.g. making each question a sub heading, or using bullets?

Reply:
Discussed. Not worth it at this time; different dialects of markdown define different things, and definition lists is not available in githubs markdown flavor *for readmes* iiuc. :)